release.mk

```diff
@@ -268,6 +271,7 @@ PDF_DOC_FILE := $(abspath $(ROOT)/toolchain/doc/_build/latex/GNU_Toolchain_for_A
 ECLIPSE_VERSION := neon-3
 ECLIPSE_VANILLA_WIN_ZIP := eclipse-cpp-$(ECLIPSE_VERSION)-win32.zip
 ECLIPSE_VANILLA_LINUX_TGZ := eclipse-cpp-$(ECLIPSE_VERSION)-linux-gtk-x86_64.tar.gz
+ECLIPSE_VANILLA_MACOS_TGZ := eclipse-cpp-$(ECLIPSE_VERSION)-macosx-cocoa-x86_64.tar.gz
```

Comment:
Our /slowfs/ru20arcgnu1/arcgnu_verif/gnu_releases/3rd-party folder contains Eclipse Oxygen for macOS as a dmg file. Could you please download it as a tar.gz there?

Reply:
Done.